<fix>[zwatch]: finish single enum rename and genarate

zstack-robot-2 opened the pull request:
Resolves: ZSTAC-76359

Change-Id: I6877636b7764616e69646d766277646d6d677268

sync from gitlab !8086

Commit:
NoLo'g'ging<fix>[storage]: Rename enum Behavior to LogMaskBehavior in annotation NoLogging

ZQLImpact

Resolves: ZSTAC-76450

Change-Id: I65707371637a7776736f6866707568756f656479

Commit:
<fix>[local]: rename enum Capability in VmInstanceConstant and VolumeConstant

1.rename enum Capability to VolumeCapability in VolumeConstant
2.rename enum Capability to VmCapability in VmInstanceConstant

Resolves: ZSTAC-76353

Change-Id: I6463666b72686c7769646c7169766c6e72617868

Commit:
<fix>[compute]: rename enum HostMaintenancePolicy

rename Enum HostMaintenancePolicy to HostMaintenanceVmOperationPolicy in HostMaintenancePolicyExtensionPoint

Resolves: ZSTAC-76356

Change-Id: I6276767169786c74717a61776363697772616977

Commit:
<fix>[host]: rename Enum Opaque to BackupStorageErrorOpaque in BackupStorageErrors

Resolves: ZSTAC-76358

Change-Id: I69647665626c6378766c73766677696a6c696f6e

Commit:
<fix>[storage]: rename enum Operation

1.Operation to SnapshotGroupOperation in VolumeSnapshotGroupOperationValidator
2.Operation to TransactionalOperation in TransactionalCallback

Resolves: ZSTAC-76360

Change-Id: I616a6569626d7364646d64767667646f68666576

Commit:
<fix>[compute]: rename Enum ReconnectDecision

1.rename enum ReconnectDecision to HostReconnectDecision in HostTrackImpl
2.rename enum ReconnectDecision to BackupStorageReconnectDecision in BackupStoragePingTracker

Resolves: ZSTAC-76366

Change-Id: I646a76626b69706275726c676263716f71686e74

Commit:
<fix>[storage]: rename Enum Type

1.rename Enum Type to ApiPathType in ApiPathTracker
2.rename Enum Type to  LogMaskType in NoLogging

APIImpact

Resolves: ZSTAC-76367

Change-Id: I70776a6f6b716c6a6c6461666f616a7472796b6d

Commit:
<fix>[zwatch]: finish single enum rename and genarate

Resolves: ZSTAC-76359

Change-Id: I6877636b7764616e69646d766277646d6d677268

Commit:
<fix>[virtualRouterProvider]: rename enum Param

Resolves: ZSTAC-76362

Change-Id: I6b7772757378637366766b727874756a65636e73

Commit:
<fix>[compute]: rename enum Params

Resolves: ZSTAC-76364

Change-Id: I746f6769716875627a61706e6f6f6f636f6a7470

zstack-robot-1 commented on testlib/src/main/java/org/zstack/testlib/ApiPathTracker.java:
```diff
@@ -25,13 +25,13 @@ public class ApiPathTracker {
 
     final List<Path> paths = new ArrayList<>();
 
-    private enum Type {
+    private enum ApiPathType {
         Message,
         HttpRPC,
     }
 
     private static class Path {
-        Type type;
+        ApiPathType type;
```
**Comment from weiqiang.xu:**

莫须有

zstack-robot-2 commented on sdk/src/main/java/org/zstack/sdk/zwatch/datatype/EventEmergencyLevel.java:
```diff
@@ -0,0 +1,8 @@
+package org.zstack.sdk.zwatch.datatype;
+
+public enum EventEmergencyLevel {
+	Emergent,
+	Important,
+	Normal,
+	Recovery,
+}
```
**Comment from weiqiang.xu:**

原先遗留，不更改

Commit:
<fix>[zwatch]: Restore enum due to StabilityTest

1.CdpTaskEventLabelNames to EventLabelNames
2.VmLabelNames to LabelNames
3.LabelMatchOperator to Operator

Resolves: ZSTAC-76359

Change-Id: I7474776c706f676972776a746b7665656e737a70
